docs/devenv_setup.md

```diff
@@ -1,163 +1,149 @@
 # MicroShift Development Environment
-The development environment bootstrap and configuration procedures are automated as described in the [Automating Development Environment Setup](./devenv_setup_auto.md) document.
-It is recommended to review the current document and use the automation instructions to create and configure the environment.
+This document describes how to setup the MicroShift development environment.
 
 ## Create Development Virtual Machine
 Start by downloading one of the supported boot images for the `x86_64` or `aarch64` architecture:
 * RHEL 9.2 from https://developers.redhat.com/products/rhel/download
 * CentOS 9 Stream from https://www.centos.org/download
 
 ### Creating VM
-Create a RHEL virtual machine with 2 cores, 4096MB of RAM and 50GB of storage.
-> Visual Studio Code may consume around 2GB of RAM. For running the IDE on the development virtual machine, it is recommended to allocate at least 6144MB of RAM in total.
+Log into the hypervisor host and run the following commands to create a RHEL virtual
+machine with 4 cores, 4096MB of RAM and 50GB of storage.
```

Comment:
Isnt manage_vm script defaulting to 8G of memory? https://github.com/openshift/microshift/blob/main/scripts/devenv-builder/manage-vm.sh#L67